README.md

```diff
@@ -372,3 +375,14 @@ The same as `cabal update`, but limited to retrieving a new index
 tarball at most once per day. This is useful for scripting CI build bots
 to prevent them from wasting time updating from Hackage when there are
 likely no interesting changes.
+
+
+System configuration
+--------------------
+
+Mafia expects both GHC and Cabal to be installed and on the `PATH`.
+
+Follow the guides below to configure your system correctly:
+
+- [Installing GHC](https://github.com/ambiata/mafia/blob/master/doc/ghc.md)
+- [Installing Cabal](https://github.com/ambiata/mafia/blob/master/doc/cabal.md)
```

Comment:
It's an anchor inside the ghc doc, so you would find it by clicking on this link, that's close enough for me